pennylane/math/quantum.py

```diff
@@ -733,6 +733,80 @@ def _compute_mutual_info(
     return vn_entropy_1 + vn_entropy_2 - vn_entropy_12
 
 
+# pylint: disable=too-many-arguments
+def vn_entanglement_entropy(
+    state, indices0, indices1, base=None, check_state=False, c_dtype="complex128"
+):
+    r"""Compute the Von Neumann entanglement entropy between two subsystems in a given state
+
+    .. math::
+
+        S(\rho_A) = -\text{Tr}[\rho_A log \rho_A] = -\text{Tr}[\rho_B log \rho_B] = S(\rho_B)
+
+    where :math:`S` is the von Neumann entropy, and :math:`\rho_A = \text{Tr_B}[\rho_{AB}]` and
+    :math:`\rho_B = \text{Tr_A}[\rho_{AB}]` are the reduced density matrices for each partition.

```

Comment:
I don't know if this was intentional, but perhaps you meant to write `\text{Tr}_A` instead of `\text{Tr_A}` etc.